lib/src/sync_worker_loop.dart

```diff
@@ -19,6 +19,8 @@ abstract class SyncWorkerConnection {
 }
 
 /// Persistent Bazel worker loop.
+///
+/// Extend this class and implement the `performRequest` method.
 abstract class SyncWorkerLoop {
```

Comment:
What's the motivation for having both a sync and an async worker loop?  It seems like duplication of effort--my expectation is that the async worker look should be sufficient for all clients.